packages/module-utils/test/test-kit.ts

```diff
@@ -49,3 +50,15 @@ export function moduleFactoryTestKit(
         evalStylableModule,
     };
 }
+
+function addStylableRuntimeToMemFs(fs: IMemFileSystem) {
```

Comment:
This method should not be coupled to `IMemFileSystem` if it only use `ensureDirectorySync` and `writeFileSync`.

Reply:
it's a local function that is called from a single place, I prefer not to complicate the type